Actualización ontología

abasco21 opened the pull request:
-	guardo lo anterior en la carpeta –anterior
-	creo una nueva ontología simplificada: o	Elimino atributos que no utilizamos.
o	todos los atributos son string salvo accesibilidad sí o no. o	Entidades: barrio, distrito y parada de metro.
o	Heredamos de PublicBuilding lgdo:http://linkedgeodata.org/ontology/PublicBuilding

abasco21 committed:
Actualización ontología

-	guardo lo anterior en la carpeta –anterior
-	creo una nueva ontología simplificada:
o	Elimino atributos que no utilizamos.
o	todos los atributos son string salvo accesibilidad sí o no.
o	Entidades: barrio, distrito y parada de metro.
o	Heredamos de PublicBuilding lgdo:http://linkedgeodata.org/ontology/PublicBuilding

abasco21 committed:
actualizado ontology

gabyarte commented on HandsOn/Group21/ontology/ontologyv4.ttl:
```diff
@@ -0,0 +1,122 @@
+neighborhood:MadridNeighborhood rdf:type owl:Class ;
+	rdfs:label "Madrid Neighborhood" .
+
+### district:MadridDisctirct
+district:MadridDisctirct rdf:type owl:Class ;
+	rdfs:label "Madrid Disctirct" .
```
There is a typo here. Instead of `MadridDisctirct`, write `MadridDistrict`

abasco21 committed:
ultima ontología

abasco21 committed:
openrefine tareas

generados:
- 200342-0-centros-dia-updated.csv
- 200342-0-centros-dia-updated-with-links.csv
- 200342-0-centros-dia-updated.json
- 200342-0-centros-dia-updated-vith-links.json
